src/renderers/native/__tests__/ReactNativeMount-test.js

```diff
@@ -1,201 +0,0 @@
-/**
```

Comment:
Was this test deleted intentionally? It looks valuable to me and tests public API.